packages/backend/migrations/20211022210203_create_invoices_table.js

```diff
@@ -1,8 +1,9 @@
 exports.up = function (knex) {
   return knex.schema.createTable('invoices', function (table) {
     table.uuid('id').notNullable().primary()
-    table.uuid('accountId').notNullable()
-    table.foreign('accountId').references('accounts.id').onDelete('CASCADE')
+
+    // Tigerbeetle account id
+    table.uuid('tbAccountId').notNullable()
```

Comment:
I'm looking ahead to renaming `paymentPointerId`s to some kind of account and wondering if we could ditch `tbAccountId` columns altogether and use peer/invoice/outgoing payment ids themselves for their corresponding tigerbeetle accound ids.